docs/iam.rst

```diff
@@ -443,7 +442,7 @@ To: ::
       ]
   }
 
-In case you are using awsbatch as a scheduler:
+When using awsbatch as a scheduler:
```

Comment:
awsbatch --> AWS Batch to be aligned with the previous list.

Reply:
Done.